### DISCUSSION TITLE
About pages content in german.

### DISCUSSION THREAD

**Discussion by jh-RLI** [openego/oeplatform#1578] [Feedback]
Provide the about pages in german language

**Comment by florianhinze** [comment]
Deutsche Version:

{% extends "base/base.html" %}
{% load django_bootstrap5 %}
{% block main-content-body %}

{% load static %} {# Important for using the *static* tag #}

<h2>Open Energy Familie und Open Energy Plattform</h2> <br>

<a href="{% static 'img/about/OpenEnergyFamily_GroupPhoto.png' %}">
    <img style="width:50%" class="mx-auto d-block" src="{% static 'img/about/OpenEnergyFamily_GroupPhoto.png' %}"
        alt="Open Energy Family" /></a>


    <h3 id="general-introduction">Allgemeine Einleitung</h3>
    <p>Ziel der Open Energy Familie ist es, Qualität, Transparenz und 
        Reproduzierbarkeit in der Energieforschung sicherzustellen. Es ist 
        eine Sammlung von <a href="https://github.com/OpenEnergyPlatform/organisation">
        verschiedenen Werkzeugen und Informationen</a> die dabei helfen, 
        mit energiebezogenen Daten zu arbeiten. Es ist eine gemeinschaftliche
        Anstrengung, alles ist offen entwickelt und wird ständig weiterentwickelt. 
        </p>
    <p>Das Hauptmodul ist die Open Energy Plattform (OEP), eine Webschnittstelle
         für den Zugang zu den meisten Modulen, insbesondere zur Community
        Datenbank.
        Es bietet die Möglichkeit, Daten mit einer Dokumentation
         (Metadaten) zu veröffentlichen und sie mit dem Quellcode und den 
         zugrundeliegenden Annahmen zu verbinden.
        </p>
    <p>Um das Problem verschiedener Namen und Datenbankformate zu lösen, wurde die
        <a href="https://openenergy-platform.org/ontology">Open Energy Ontologie
            (OEO)</a>
        entwickelt. Diese Ontologie stellt ein kontrolliertes Vokabular mit 
        klaren Definitionen von Ausdrücken und deren Beziehungen zueinander bereit.
        Die Anmerkungen zu den Daten bieten die Möglichkeit der erweiterten 
        Suche und des logischen Abfragens über Datensätze hinweg.        
    </p>


    <h3 id="what-energy-system-modelling">Was? Energiesystemmodellierung und
        Daten</h3>
    <p>Offen lizenzierte Daten, die im Bereich der Klima- und Energieforschung relevant sind, können
        hier veröffentlicht werden. Insbesondere Daten und Szenarien zur 
        Energiesystemmodellierung sowie weitere Informationen zu den zugrunde liegenden
        Modellen, Rahmenwerken und Studien können hier gefunden werden.
        </p>
    <p>Energiesystemmodelle werden verwendet, um das aktuelle und zukünftige 
        Energiesystem zu erforschen und werden häufig bei Fragen zum Ausbau von
        erneuerbaren Energien und Energiespeichern sowie der Energie- und 
        Klimapolitik angewendet. Die Modelle selbst variieren stark in 
        Bezug auf ihre Art, ihren Aufbau, ihre Programmierung,
        Anwendung, Umfang, Detaillierungsgrad, Ausgereiftheit und Unzulänglichkeiten.
        </p>
    <p>Die OEP betrachtet den gesamten Modellierungsprozess von der Vorverarbeitung der
        Rohdaten, den Simulationen und der Nachbearbeitung der Ergebnisdaten. Das Herzstück
        der OEP ist die zugrunde liegende
        <a href="https://openenergy-platform.org/dataedit/schemas">Open Energy
            Datenbank (OEDB)</a>, die von jedem genutzt werden kann und die von 
            einem Team von Entwicklern von verschiedenen Instituten entwickelt wird.
    </p>


    <a href="{% static 'img/about/OpenEnergyFamily_modeling-cycle.png' %}">
        <img style="width:50%" class="mx-auto d-block"
            src="{% static 'img/about/OpenEnergyFamily_modeling-cycle.png' %}" alt="OEP circle simple" />
    </a>

    <h3 id="how-open-science">Wie? Open Science</h3>
    <p>Wir folgen dem Prinzip von 
        <a href="https://en.wikipedia.org/wiki/Open_science">Open Science</a>
        und fördern es. Wir bieten eine offene Infrastruktur für die Energiesystemforschung an.
        Der Schwerpunkt liegt auf Open Data und der Bereitstellung von Modellierungsergebnissen 
        unter offenen Lizenzen. Die OEP nutzt ein umfangreiches Metadatenset, das
        <a href="https://github.com/OpenEnergyPlatform/oemetadata">OEMetadata</a>.
        Es basiert auf dem
        <a href="https://frictionlessdata.io/specs/tabular-data-package/">Tabular
            Data Package</a>
        Spezifikationen und den
        <a href="https://www.go-fair.org/fair-principles/">FAIR 
            (Findability, Accessibility, Interoperability, Reuse) Prinzipien</a>
        um die Auffindbarkeit, den Zugang, die Interoperabilität und die Wiederverwendbarkeit
        von digitalen Vermögenswerten sicherzustellen.
    </p>


    <a href="{% static 'img/about/OpenEnergyFamily_modeling-open-science.png' %}">
        <img style="width:50%" class="mx-auto d-block"
            src="{% static 'img/about/OpenEnergyFamily_modeling-open-science.png' %}" alt="OEP circle simple" />
    </a>

    <h3 id="who-oep-community">Wer? Die OEP-Gemeinschaft</h3>
    <p>The Open Energy Family is a cross-project research project that is
        continuously being developed by
        various universities and institutes. The source code of all members of the
        family is
        open source and accessible on <a href="https://github.com/OpenEnergyPlatform/">GitHub</a>.
        Get involved and contribute!</p>

    <h3 id="Supported-By">Supported By</h3>
    <img style="width:25%" class="mx-auto d-block" src="{% static 'logos/funder/BMWK.png' %}"/>

    <h4 id="who-projects">Research Projects</h4>
    <div class="d-flex justify-content-between flex-md-wrap">
        {% for project in projects %}
        <a class="logo" href="{% url 'project_detail'  project_id=project.id %}">
            <img class="flex-item" src="{% static 'logos/project/' %}{{project.logo}}" alt="{{project.id}}" width="275"
                border="0" />
        </a>
        {% endfor %}
    </div>

    <h4 id="who-partners">Partners</h4>
    <div class="d-flex justify-content-between flex-md-wrap">
        <a class="logo" href="http://iks.cs.ovgu.de/IKS.html" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/OvGU.png' %}"
                alt="Otto von Guericke Universität Magdeburg - IKS" width="245" height="85" border="0" />
        </a>
        <a class="logo" href="http://reiner-lemoine-institut.de/en/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/RLI.png' %}" alt="Reiner Lemoine Institut" width="140"
                height="102" border="0" />
        </a>
        <a class="logo" href="https://www.uni-flensburg.de/en/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/EUF.png' %}" alt="Europa-Universität Flensburg"
                width="250px" />
        </a>
        <a class="logo" href="http://www.znes-flensburg.de/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/ZNES.png' %}"
                alt="Zentrum für nachhaltige Energiesysteme" width="250px" />
        </a>
        <a class="logo" href="https://www.dlr.de/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/DLR-VE.png' %}" alt="DLR-VE (NEXT Energy)" width="268"
                height="109" border="0" />
        </a>
        <a class="logo" href="https://www.hereon.de/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/partner_logo_hereon.png' %}" alt="HZG-Hereon"
                width="268" height="109" border="0" />
        </a>
        <a class="logo" href="https://www.iee.fraunhofer.de/en.html" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/IEE.png' %}" alt="Fraunhofer IEE" width="250px" />
        </a>
        <a class="logo" href="https://www.oeko.de/en/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/OEI.png' %}" alt="Öko Institut" width="250px" />
        </a>
        <a class="logo" href="https://www.fz-juelich.de" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/FZJuelich-1.jpg' %}" alt="FZJ" width="250px" />
        </a>
        <a class="logo" href="https://infai.org/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/InfAI-Logo.png' %}" alt="InfAI" width="250px" />
        </a>
        <a class="logo" href="https://www.pik-potsdam.de/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/pik_logo.png' %}" alt="PIK" width="250px" />
        </a>
        <a class="logo" href="https://www.ier.uni-stuttgart.de/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/Logo_IER_UniStuttgart-1200x386.png' %}" alt="IER"
                width="250px" />
        </a>
        <a class="logo" href="https://www.fortiss.org/" target="_blank">
            <img class="flex-item" src="{% static 'logos/partner/partner_logo_fortiss.png' %}" alt="fortiss"
                width="250px" />
        </a>
    </div>
    {% endblock %}
